vcd/helpers_test.go

```diff
@@ -0,0 +1,63 @@
+// +build vapp vm ALL functional
```

Comment:
proposal: helpers_test.go -> custom_terraform_assertions_test.go

Reply:
Ok, casting a vote here:
* testcheck_funcs_test.go
* custom_terraform_assertions_test.go

cc @lvirbalas @dataclouder

Reply:
I am ok with both ;-)

Reply:
Maybe a third variant:
* custom_assertions_test.go

p.s. Just noticed a typo in comment:
`accross -> across`

Reply:
> p.s. Just noticed a typo in comment:
> `accross -> across`

Thanks. Fixed.

Reply:
After further discussion, OK with `testcheck_funcs_test.go`, because:
1. It has custom TestCheckFunc inside
2. It is consistent with `suppress_funcs.go` and `validate_funcs.go`